fhir-server/liberty-config/server.xml

```diff
@@ -99,6 +99,12 @@
     </dataSource>
 
     <webAppSecurity allowFailOverToBasicAuth="true" singleSignonEnabled="false"/>
+    
+    <cors domain="/"
+        allowedOrigins="*"
+        allowedMethods="GET, DELETE, POST, PUT, HEAD"
+        allowedHeaders="*"
```

Comment:
I'm not sure about the wildcard here, I think it's a-ok to have '*'

Reply:
Its allowed per the liberty docs.  Not sure if there are security ramifications or not, but I didn't see any reason to restrict which headers are allowed at this time.